docs/man/ntp.toml.5.md

```diff
@@ -14,61 +14,34 @@ allowing a json-like syntax.
-that concept. Most NTP traffic, especially across the public internet, almost
+that idea. Most NTP traffic, especially across the public internet, almost
 exclusively uses this mode, so it is not considered a practical limitation for
 most scenarios.
 
-# CONCEPTS
-
-## Source
-A source represents a set of one or more connections to time sources (another
-NTP server) where the daemon retrieves time information from. The time
-information from these sources are then filtered and combined to update the
-local clock. The daemon can have multiple of these sources sets configured to
-allow for a wider range of time sources. When ntp-daemon is used in a client
-only setup the configuration only contains source configurations and no server
-configurations.
-
-## Server
-A server distributes time information to clients who request it. A server
-listens on a single socket address for client packets and responds to them. The
-daemon can listen on multiple sockets by creating multiple server
-configurations. Generally, when NTP is configured as a server it also has one
-or more source configurations from where time is retrieved. The daemon currently
-does not have support for local time sources, if no sources are configured, then
-another process should be in place to discipline the local system clock.
-
-## NTS and keysets
-NTS, Network Time Security, is a protocol that uses a TLS handshake to
-exchange secrets to allow verifying that responses from an NTP server have not
-been tampered with. The daemon supports running NTS in both server and client
-(source) connections. For the server functionality a key exchange server also
-needs to be configured. Currently only an internal key exchange server is
-supported. The NTP server and TLS key exchange server of the ntp-daemon both
-need to be aware of a shared set of keys, these keys are called the keyset.
-
-## Logging and observability
-The daemon can log messages about how it is operating in different logging
-levels. These log messages come in several priorities (from low to high):
-*trace*, *debug*, *info*, *warn* and *error*. You can use these log messages to
-monitor the daemon. The log messages are outputted on stdout.
-
-Aside from logging, you can use the observability metrics to get semi-realtime
-information about several key metrics for both the client and server part of
-the daemon. These metrics are exposed via the `ntp-ctl` CLI utility, or by using
-our prometheus / openmetrics exporter.
-
-## Clocks and synchronization
-Of course the main reason for running an NTP daemon is to get accurate time
-information. This should result in the daemon adjusting your system clock. The
-algorithm that is used to combine the information from several sources and
-update the system clock can be configured using the synchronization section in
-the configuration file.
+# SOURCE MODES
+Sources (see the section below for details) can be configured any of the
+following modes:
```

Comment:
Suggest we phrase this more along the lines of types of sources that we support.